src/Plugin/GraphQL/DataProducer/EntityDefinition/EntityDefinition.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Drupal\graphql\Plugin\GraphQL\DataProducer\EntityDefinition;
+
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Entity\EntityTypeManager;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\graphql\GraphQL\Execution\FieldContext;
+use Drupal\graphql\Plugin\GraphQL\DataProducer\DataProducerPluginBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Gets entity definition for a given entity type.
+ *
+ * @DataProducer(
+ *   id = "entity_definition",
+ *   name = @Translation("Entity definition"),
+ *   description = @Translation("Return entity definitions for given entity type."),
+ *   consumes = {
+ *     "entity_type" = @ContextDefinition("string",
+ *       label = @Translation("Entity type")
+ *     ),
+ *     "bundle" = @ContextDefinition("string",
+ *       label = @Translation("Bundle"),
+ *       required = FALSE
+ *     ),
+ *     "field_types" = @ContextDefinition("FieldTypes",
+ *       label = @Translation("Field types"),
+ *       default = "ALL",
+ *       required = FALSE
+ *     )
+ *   },
+ *   produces = @ContextDefinition("any",
+ *     entity_definition = @Translation("Entity definition")
```

Comment:
The "label" property on the annotation for all the "produces" contexts in this PR are wrong. I assume you copy pasted them and did a string serach and replace all. Should be "label".

Reply:
fixed